src/commonmark/en/content/developer/web-api/metadata-gist.md

```diff
@@ -0,0 +1,732 @@
+* `pluck(<field>)` => (array of) `string`: extract single text property of the 
+  object or of each collection item
+
+A field can receive both the `rename` transformer and one of the other 
+transformers, for example:
+
+    /api/organisationUnits/gist?fields=*,children::size~rename(child-count)
+
+The returned items now no longer have a `chilren` member but a `child-count`
+member instead. Note that `rename` also affects the member name of the URI
+reference given in `apiEndpoints`.
+
+
+## Synthetic Fields
+<!--DHIS2-SECTION-ID:gist_syntheticFields-->
+
+The `/gist` API is tightly coupled to properties that exist the database.
+This means properties that aren't stored in the database usually aren't 
+available.
+The exception to this are the "synthetic" properties which are dynamically 
+computed on the basis of one or more database stored properties.
+
+Synthetic properties are available for all endpoints where the persisted 
+properties needed to compute the synthetic property exist.
+
+Except for the `apiEndpoints` property which is automatically added when needed 
+all other synthetic properties are not included by default and have to be 
+requested explicitly in the list of `fields`.
+
+
+### Overview
+Synthetic fields in alphabetical order:
+
+* `apiEndpoints`: contains links to browse nested complex objects or collections
+* `href`: link to the list item itself (single item view)
+* `displayName`: translated `name` (always translated)
+* `displayShortName`: translated `displayName` (always translated)
+
+
+### The `href` Field
+<!--DHIS2-SECTION-ID:gist_syntheticFields_href-->
+
+Each item in a `/gist` response can link to itself. This link is given in the 
+`href` property.
+
+To add the `href` field use (for example):
+
+		/api/<object-type>/gist?fields=*,href
+
+
+### The `displayName` and `displayShortName` Field
+<!--DHIS2-SECTION-ID:gist_syntheticFields_displayName-->
+
+By definition the `displayName` is the translated `name` and the 
+`displayShortName` is the translated `shortName`. 
+
+To add `displayName` or `displayShortName` add it to the list use (for example):
+
+		/api/<object-type>/gist?fields=*,displayName
+		/api/<object-type>/gist?fields=*,displayShortName
+
+Note that by default all translatable properties like `name` and `shortName` 
+would also be translated. When `translate=false` is used to disable this 
+`displayName` and `displayShortName` stay translated.
+
+
+### The `apiEndpoints` Field
+<!--DHIS2-SECTION-ID:gist_syntheticFields_apiEndpoints-->
+
+This property provides the links to further browse complex objects or list of 
+items that are included in the `/gist` response in form of a transformed simple 
+value like an item count.
+
+The `apiEndpoints` object will have a member of the same name for every member 
+in the item that was transformed to a simple value.
+
+For example, `/api/users/gist?fields=id,userGroups::size,organisationUnits::size` 
+returns items in the form:
+
+```json
+	{
```

Comment:
I think we use two spaces for indent for JSON in docs, a bit arbitrary of course, would be good to have it consistent.

Reply:
Configured this now in IntelliJ and formatted with it - works smoothly.